hack/operatorhub/config.yaml

```diff
@@ -1,6 +1,6 @@
-newVersion: 1.2.1
-prevVersion: 1.2.0
-stackVersion: 7.8.1
+newVersion: 1.3.0
+prevVersion: 1.2.1
+stackVersion: 7.9.0
```

Comment:
7.9.1 has been released but it will surely be 7.10.0 :crossed_fingers:.

Reply:
I'll bump to 7.9.1 for now as it's the currently available version. Hopefully the release manager will double check the values before cutting the next release :)